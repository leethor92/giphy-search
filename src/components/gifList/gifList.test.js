import React from 'react';
import { shallow } from 'enzyme';
import GifList from './index';
describe('GifList', () => {
    it('should render correctly in mode', () => {

        let gifs = [
            {
                id: 1
            },
            {
                id: 2
            },
            {
                id: 3
            }
        ]

        const component = shallow(<GifList gifs={gifs} />);

        expect(component).toMatchSnapshot();
    });
});