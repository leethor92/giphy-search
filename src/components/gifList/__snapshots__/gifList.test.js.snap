// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifList should render correctly in mode 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <Index
      gif={
        Object {
          "id": 1,
        }
      }
      key="1"
    />
    <Index
      gif={
        Object {
          "id": 2,
        }
      }
      key="2"
    />
    <Index
      gif={
        Object {
          "id": 3,
        }
      }
      key="3"
    />
  </div>
</div>
`;
